dev/io.openliberty.netty.internal.impl/src/io/openliberty/netty/internal/tcp/TCPMessageConstants.java

```diff
@@ -1,14 +1,11 @@
 /*******************************************************************************
- * Copyright (c) 2021 IBM Corporation and others.
+ * Copyright (c) 2021, 2022 IBM Corporation and others.
```

Comment:
We should probably discuss this a bit more. Last discussion we had about this I planned on updating everything to the latest year (2023) but since the files/changes were introduced beforehand and not updated (only rebased commits, etc) then we decided to only update the ones that actually had changes.